packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -60,162 +70,221 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = ({
   const essentialState = useEssentialTemplates();
 
   const isSyncedSandboxesPage = location.pathname.includes('/synced-sandboxes');
-  const isDashboardPage = location.pathname.includes('/dashboard');
   const isUser = user?.username === activeTeamInfo?.name;
 
+  /**
+   * Checking for user because user is undefined when landing on /s/, even though
+   * hasLogIn is true.
+   */
+  const showTeamTemplates = hasLogIn && user;
+
   const tab = useTabState({
```

Comment:
nitpick: I would name this tabState maybe? the `...tab` destructuring below can be confusing, especially since it is passed to multiple components

Reply:
Good idea!